NEWS.md

```diff
@@ -12,6 +12,7 @@
 * Updated the `get_varlabels` method for `FilterStates` classes. It now accepts a vector input.
 * Exported `S3` generic function `init_filter_states` so that it can be used in other packages.
 * Added a `FilterPanelAPI` class to encapsulate the API of a filter panel.
+* Improved filter state presentation in `FilterState$format`.
```

Comment:
News entry under already released version? Interesting 🤔 

Please create a new header with the new version and put it there

Reply:
Over the # teal.slice 0.2.0 please add # teal.slice 0.2.0.9000 and add there the new record inside ### New features

Reply:
We should update the DESCRIPTION file Version to 0.2.0.9000 too.
This one is needed if we will be the first merged PR after the release.